libs/deserializer/src/test/java/de/cyface/deserializer/BinaryFormatDeserializerTest.java

```diff
@@ -184,6 +206,210 @@ void test() throws IOException, InvalidLifecycleEvents, UnsupportedFileVersion {
         }
     }
 
+    /**
+     * This test generates random sensor values within the supported value range and deserializes the bytes to ensure
+     * the values did not change.
+     */
+    @DisplayName("Happy Path test for the serialization and deserialization of 3d points.")
+    @Test
+    void testSerializeDeserialize() throws IOException, InvalidLifecycleEvents {
+
+        // Arrange - Events: start, stop (1 track)
+        final var batches = 100;
+        final var sensorPointsPerBatch = 100;
+        final var sensorPoints = batches * sensorPointsPerBatch;
+        final long measurementStart = 1_660_000_000_000L;
```

Comment:
If you assign a long constant, this should become a long var even without the datatype, doesn't it?

Reply:
Was a typo, thanks :)